test/rabbit_stressor_test.js

```diff
@@ -16,17 +16,27 @@ suite('Rabbit Stressor', () => {
     }
   });
 
-  test('sendStringMessages', async () => {
+  const sendStringMessages = async (delayBetweenMessages) => {
     const queueName = 'stress queue';
     const messages = ['some', 'string', 'messages', 'being', 'sent'];
 
     await helper.rabbit.createQueue(queueName);
-    helper.stressor.sendMessages(queueName, messages);
+    await helper.stressor.sendMessages(queueName, messages, delayBetweenMessages);
 
     const messagesFromQueue = await helper.rabbit.messagesFromQueue(queueName);
     assert(messagesFromQueue.length > 0, 'Did not receive any messages from queue');
 
     await helper.rabbit.deleteQueue(queueName);
+  };
+
+  test('sendStringMessagesNoDelay', async () => {
+    const delayBetweenMessages = 0;
+    await sendStringMessages(delayBetweenMessages);
```

Comment:
I `await` because I think the tests should to execute synchronously. For example, if we do not `await` in these test cases, then we can have a race condition when awaiting for the internal routines of `sendStringMessages` to complete. By this, I mean that we will have both test cases executing at the same time, reading and writing from the same queue which results in undefined behaviour and failed tests. I also tried running both of the test cases without `await` and they fail.